lib/constants.dart

```diff
@@ -1,7 +1,15 @@
-const ALARMS = 'alarms';
-const ALARM_SETS = 'alarm_sets';
-const FOCUS_DURATION_KEY = 'focus_duration';
-const BREAK_DURATION_KEY = 'break_duration';
-const FROM_TIME_OF_DAY_KEY = 'from_date';
-const TO_TIME_OF_DAY_KEY = 'to_date';
-const ALARM_DATABASE = 'alarm_database';
+/// Database constants
+const alarmDatabase = 'alarm_database';
+const alarms = 'alarms';
+const alarmSets = 'alarm_sets';
+
+/// Create new alarm constants
+const focusDurationKey = 'focus_duration';
+const breakDurationKey = 'break_duration';
+const fromTimeOfDayKey = 'from_date';
```

Comment:
U produccovom constants.dart imenujemo visiDomenNiziDomen...
```suggestion
const keyTimeOfDayFrom = 'from_date';
```
i sortiramo alfabetski (unutar jedne sekcije). Boli te jajo, a ako te ne boli uopste, mozes cak baciti pogled na constants.dart tamo kao sablon.

Reply:
Ove konstante cu ukloniti u jednom momentu jer su neke suvisne, bice pomlacene nakon refaktora "Edit screena", za koji moram napraviti tiket.

Reply:
Ne mogu sad da ih peglam jer cu ih svakako pobrisati i srediti eventualno, a da bih to uradio sad morao bih da radim i edit ekran a to mi se ne da hahah